bank-api-library/LICENSE.md

```diff
@@ -1 +1 @@
-Moved to http://developer.gini.net/gini-mobile-android/bank-api-library/html/license.html
\ No newline at end of file
+Moved to http://developer.gini.net/gini-mobile-android/bank-api-library/library/html/license.html
```

Comment:
I'm trying to use " https" in all links where i can notice it ( warning not secure looks a bit disturbing)😊

Reply:
That's a great point! I will replace all links to use https.

Reply:
I updated it in PR #11.